collectors/UcsmChassisFaultCollector.py

```diff
@@ -0,0 +1,30 @@
+from prometheus_client.core import GaugeMetricFamily
+from BaseCollector import BaseCollector
+
+
+class UcsmChassisFaultCollector(BaseCollector):
+    def __init__(self, creds, config):
+        super().__init__(creds, config)
+
+    def describe(self):
+        yield GaugeMetricFamily("ucsm_chassis_metrics", "ucsm_chassis_fault_collector")
+
+    def collect(self):
+        print("UcsmChassisFaultCollector: Get updated handles !")
+        self.get_handles()
+        g = GaugeMetricFamily('ucsm_chassis_faults', 'UCSM server chassis faults',
```

Comment:
metric name in describe and collect should be the same. Please correct it for all other occurrences of your collectors too.

Reply:
Changed for all.